code/modules/atmospherics/machinery/portable/scrubber.dm

```diff
@@ -163,6 +163,7 @@
 
 	var/movable = FALSE
 	use_overlays = FALSE
+	biomass_produce = BIOMASS_HARVEST_MEDIUM
```

Comment:
I am not sure about this one since it's not really related to biomass

Reply:
It's based on the 1.0 list, the huge scrubbers were a harvestable source.